README.md

```diff
@@ -1,20 +1,21 @@
 # dat
 
-Dat is a decentralized data tool for distributing datasets small and large.
+Dat is a decentralized data tool for distributing data small and large.
 
 [![#dat IRC channel on freenode](https://img.shields.io/badge/irc%20channel-%23dat%20on%20freenode-blue.svg)](http://webchat.freenode.net/?channels=dat)
 [![datproject/discussions](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/datproject/discussions?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[![docs](https://readthedocs.org/projects/pip/badge/?version=latest)](http://dat-data.readthedocs.org)
+[![docs](https://readthedocs.org/projects/dat-cli/badge/?version=latest)](http://dat-data.readthedocs.org)
 
 
 Windows        | Mac/Linux
 -------------- | ------------
 [![Build status](https://ci.appveyor.com/api/projects/status/github/maxogden/dat?branch=master&svg=true)](https://ci.appveyor.com/project/maxogden/dat) | [![Travis](https://api.travis-ci.org/maxogden/dat.svg)](https://travis-ci.org/maxogden/dat)
 
+### Development Status
 
-This is the Dat 1.0 pre-release candidate. We want to make Dat into a data versioning, forking, and syncing tool. The first feature set we are working on is easy file distribution.
+This is the Dat 1.0 release candidate (RC2). We are actively seeking feedback & developing this release candidate. Follow [this issue](https://github.com/datproject/projects/issues/5) for the Dat CLI road map discussion and see [known RC2 issues](https://github.com/maxogden/dat/issues/486).
 
-Key features:
+### Key features:
```

Comment:
maybe we should symlink docs/index.md to be readme.md, so we don't have to copy things over and can just edit readme.md and that's the first thing people see when they go to the generated docs page